tests/_utils/handle_error_lib.sh

```diff
@@ -40,7 +40,7 @@ function clean_table() {
 	run_sql_tidb "drop table if exists ${db}.${ta};"
 	run_sql_tidb "drop table if exists ${db}.${tb1};"
 	run_sql_tidb "drop table if exists ${db}.${tb2};"
-
+	run_sql_both_source "reset master"
```

Comment:
Any problem without this line?

Reply:
It is helpful when debugging binlog files manually

Reply:
But `reset master` rarely occurs upstream. Users mostly start a task with the previous binlog files. I think test without `reset master` cover more scenes